fcos/release-checklist.md

```diff
@@ -155,5 +160,5 @@ NOTE: In the future, most of these steps will be automated.
 
 ## Housekeeping
 
-- [ ] If one doesn't already exist, [open an issue](https://github.com/coreos/fedora-coreos-streams/issues/new?labels=kind/release,jira&template={{ stream }}.md) in this repo for the next release in this stream. Use the approximate date of the release in the title.
+- [ ] If one doesn't already exist, [open an issue](https://github.com/coreos/fedora-coreos-streams/issues/new?template={{ stream }}.md) in this repo for the next release in this stream. Use the approximate date of the release in the title.
```

Comment:
we can remove the labels parameter now that the front matter takes care of adding them